src/core/editform/qgsattributeeditorcontainer.h

```diff
@@ -52,6 +52,35 @@ class CORE_EXPORT QgsAttributeEditorContainer : public QgsAttributeEditorElement
      */
     virtual void addChildElement( QgsAttributeEditorElement *element SIP_TRANSFER );
 
+    /**
+     * Insert a child element into this container. This may be another container, a field or a relation.
+     * Note that the element must first be taken out of its original container.
+     *
+     * \param index The location to insert the child
+     * \param widget The element to add as child
+     * \since QGIS 3.26
+     */
+    virtual void insertChildElement( int index, QgsAttributeEditorElement *widget SIP_TRANSFER );
+
+    /**
+     * Take a child element from a container, so that it can be inserted into another container.
+     *
+     * \param index The index of the child element
+     * \returns widget The element to add as child
+     * \since QGIS 3.26
+     */
+    QgsAttributeEditorElement *takeChildElement( int index );
```

Comment:
I **think** this needs:

```suggestion
    QgsAttributeEditorElement *takeChildElement( int index ) SIP_TRANSFERBACK;
```

(it's either that or SIP_FACTORY, and it's a 50/50 coin toss which is correct :roll_eyes: )